.eslintrc

```diff
@@ -10,8 +10,7 @@
     }
   },
   "env": {
-    "browser": true,
-    "jest": true
```

Comment:
Why remove from here and put in the other one? Other than that things make sense.

Reply:
This is a linter config specific to `jest` so I moved it to the jest's specific config file `spec/javascripts/.eslintrc`.

However, I've looked into it and it seems this env is outdated and it belonged back when there wasn't a `eslint-plugin-jest`. It can be removed now.